packages/core/src/theme/createTheme.ts

```diff
@@ -27,10 +27,7 @@ export function createTheme() {
     const breakpoints = getSlice('breakpoints');
     if (typeof value === 'object' && breakpoints) {
       const keys = Object.keys(value);
-      return (
-        keys.includes('default') ||
```

Comment:
@mauroerta I removed this fix and everything still works fine. So, as you suggested, I removed this specific fix